src/database/models/role.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @swagger
+ *
+ * definitions:
+ *   Role:
+ *     type: object
+ *     required:
+ *       - roleName
+ *       - roleValue
+ *     properties:
+ *       roleName:
+ *         type: string
+ *       roleValue:
+ *         type: string
+ *       createdAt:
+ *         type: string
+ *       updatedAt: 
+ *         type: string
+ */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Roles = sequelize.define('Roles', {
+    roleName: { type: DataTypes.STRING, allowNull: false },
+    roleValue: { type: DataTypes.STRING,  allowNull: false },
+  }, {});
```

Comment:
change this line to `}, { freezeTableName: true });`  to avoid the table to change its name